### ISSUE TITLE
Bank Transfers

### ISSUE THREAD

**Issue by tstraus13** [alpacahq/alpaca-trade-api-csharp#369]
Hello,

I have combed through the code and do not see any methods or options to initiate/create a bank transfer. Has this been implemented and I have just missed it? It is part of the Broker API so maybe I am misunderstanding and it cannot be implemented here. Here is the link I found to their API docs for transfers.

https://alpaca.markets/docs/broker/api-references/funding/transfers/

Thanks.

**Comment by OlegRa** [comment]
@tstraus13 The [YAML file](https://github.com/alpacahq/bkdocs/blob/master/assets/openapi.yaml) for the Alpaca Broker API. I'm still thinking about possible relationships between this SDK and Alpaca Broker API. Each solution (same package, separate independent package, extracting some interfaces into the shared package) has its own pros and cons.

I'll add a separate issue for discussing this problem with the community. But in any case, these changes will be part of version 6.0 and I will do it only after implementing most of the other tasks planned for this major release.
